src/mapToRelative.js

```diff
@@ -3,20 +3,12 @@ import path from 'path';
 import { toPosixPath } from './utils';
 
 
-function resolve(cwd, filename) {
-  if (path.isAbsolute(filename)) {
-    return filename;
-  }
-
-  return path.resolve(cwd, filename);
-}
-
```

Comment:
I only removed this because it didn't make any difference and also it seems that if `filename` would be absolute, `path.resolve` would return it as-is anyway.